[Dev] Return fig object by VizroAI plot function (#441)
mckinsey/vizro: vizro-ai/docs/pages/user-guides/run-vizro-ai.md

You may prefer to integrate Vizro-AI into an application with a UI that users use to input prompts using a text field.

There are two ways to integrate Vizro-AI into an application, directly and by accessing the chart code behind a `fig` object.

> 1. Vizro-AI's `plot` method returns `plotly.graph_objects` fig object that can be directly used within `Vizro` dashboard.

Reviewer: ```suggestion
1. Vizro-AI's `plot` method returns a `plotly.graph_objects` object (`fig`) that can be used directly within a `Vizro` dashboard.
```

    !!! example "Direct application integration"

        === "app.py"
            ```py
            import vizro.plotly.express as px
            from vizro_ai import VizroAI

            vizro_ai = VizroAI()

            df = px.data.gapminder()
            fig = vizro_ai.plot(df, "describe life expectancy per continent over time")
            ```

> 2. Vizro-AI's `_get_chart_code` method returns the Python code string that can be used to prepare the data and create the visualization. This code is validated and debugged to ensure that it is executable and ready to be integrated.

Reviewer: ```suggestion
2. Vizro-AI's `_get_chart_code` method returns a string of Python code that manipulates the data and creates the visualization. Vizro-AI validates the code to ensure that it is executable and can be integrated.
```
